### PR TITLE
webworker

### PR DESCRIPTION
**Pull request by SamSaffron**
- Improve portability, use multiple patterns to load wasm
- DEV: move wasm work to web worker

### CODE REVIEW

File: assets/svgbob_worker.js [hunk 1]
```diff
@@ -0,0 +1,150 @@
+let importObject = {
```
Comment (nattsw) [inline assets/svgbob_worker.js:1]: Found plenty of `let`s here and there. Would be nice to use `const` if not mutating so I don't have think that it's going to be modified.

Comment (SamSaffron) [reply assets/svgbob_worker.js:1]: I was not 100% sure of const is going to be allowed in non transpiled code. 

I guess it is pretty green, https://caniuse.com/const will change a few around

File: assets/svgbob_worker.js [hunk 2]
```diff
@@ -0,0 +1,150 @@
...
+let wasm;
+let wasmModuleUrl = "https://unpkg.com/svgbob-wasm@0.4.1/svgbob_wasm_bg.wasm";
+
+if (WebAssembly.instantiateStreaming) {
+  WebAssembly.instantiateStreaming(
+    fetch(wasmModuleUrl),
+    importObject
+  ).then((response) => {
+    wasm = response.instance.exports;
+  })
```
Comment (nattsw) [inline assets/svgbob_worker.js:17]: I'm not sure if this case will ever happen. I suppose this is intended to be a polyfill for `instantiateStreaming`?

The else case is kind of funny. I suppose, it's doing the same thing in a different way.

Nothing that needs to be changed, though.

Comment (SamSaffron) [reply assets/svgbob_worker.js:17]: Yeah I was worried this was blowing stuff up turns out it probably was not: 

https://caniuse.com/?search=instantiatestreaming

Looks green enough to me, I may remove ... not sure.

**Review by nattsw — APPROVED** [review]

**Pull request merged**